src/app/utils/faucet-links.ts

```diff
@@ -9,6 +9,9 @@ const faucetLinks: Partial<Record<Network, Partial<Record<Layer, Partial<Record<
     [Layer.emerald]: { [Ticker.TEST]: `${faucets.oasisTestnet}?paratime=emerald` },
     [Layer.sapphire]: { [Ticker.TEST]: `${faucets.oasisTestnet}?paratime=sapphire` },
     [Layer.cipher]: { [Ticker.TEST]: `${faucets.oasisTestnet}?paratime=cipher` },
+    [Layer.pontusxdev]: {
+      [Ticker.EUROe]: `mailto:contact@delta-dao.com?subject=${encodeURIComponent('Request test tokens')}`,
+    },
     [Layer.pontusx]: {
       [Ticker.EUROe]: `mailto:contact@delta-dao.com?subject=${encodeURIComponent('Request test tokens')}`,
     },
```

Comment:
How would they know, which Layer to send the test tokens to? Wouldn't be nice to include it in the subject, for which Layer it is?